org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/contentassist/CompletionProposalRequestor.java

```diff
@@ -134,4 +140,29 @@ public void setIgnored(int completionProposalKind, boolean ignore) {
 			setRequireExtendedContext(true);
 		}
 	}
+
+	private void acceptPotentialMethodDeclaration(CompletionProposal proposal) {
+		try {
+			IJavaElement enclosingElement = null;
+			if (response.getContext().isExtended()) {
+				enclosingElement = response.getContext().getEnclosingElement();
+			} else if (unit != null) {
+				// kept for backward compatibility: CU is not reconciled at this moment, information is missing (bug 70005)
+				enclosingElement = unit.getElementAt(proposal.getCompletionLocation() + 1);
+			}
+			if (enclosingElement == null) {
+				return;
+			}
+			IType type = (IType) enclosingElement.getAncestor(IJavaElement.TYPE);
+			if (type != null) {
+				String prefix = String.valueOf(proposal.getName());
+				int completionStart = proposal.getReplaceStart();
+				int completionEnd = proposal.getReplaceEnd();
+				int relevance = proposal.getRelevance() + 4;
+
+				GetterSetterCompletionProposal.evaluateProposals(type, prefix, completionStart, completionEnd - completionStart, relevance + 2, proposals);
```

Comment:
why adding +2, when you already tweaked the relevance in the previous line? might as well add +6 there